Neos.Neos/Documentation/CreatingASite/NodeTypes/DynamicConfigurationProcessing.rst

```diff
@@ -0,0 +1,77 @@
+.. _dynamic-configuration-processing:
+
+============================================
+Dynamic Client-side Configuration Processing
+============================================
+
+.. note:: This API is rather low-level and still experimental, we might change
+   some of the implementation details or compliment it with a more high-level API.
+
+All configuration values that begin with ``ClientEval:`` are dynamically evaluated on
+the client side. They are written in plain JavaScript (evaluated with ``eval``) and
+have ``node`` variable in the scope pointing to the currently focused node, with all
+transient inspector changes applied. For now it is only related to the nodetypes
+inspector configuration, but in the future may be extended to the other parts of
+the user interface.
+
+
+A few Practical Examples
+========================
+
+Hiding one property when other is not set
+-----------------------------------------
+
+Here is an example how to hide the property ``border-color`` if ``border-width`` is empty
+by changing its group name to a non-existant value:
+
+.. code-block:: yaml
+
+  'Some.Package:NodeType':
+    properties:
+      border-width:
+        type: integer
+        ui:
+          inspector:
+            group: 'style'
+      border-color:
+        type: string
+        ui:
+          inspector:
+            group: 'ClientEval:node.properties["border-width"] ? "style" : "invalid-group"'
+
+Dependant SelectBoxes
```

Comment:
Dependant with a only exists as a noun. I think "Dependent" would be better here.
Also, I would use Select Boxes instead of SelectBoxes